### PR TITLE
#239 test get risks for project endpoint

### PR DESCRIPTION
**Pull request by chloezhong3**
## Changes

created two tests for getRisksForProject Endpoint 

## Notes

This is not the final, but we would like it to be reviewed and get feedback 

## Test Cases
- the given projectId does not correspond to a project 
- it works 

## Screenshots

Image 10-19-22 at 7 28 PM
Image 10-19-22 at 7 29 PM

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

File: src/backend/tests/risks.test.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+const app = express();
+app.use(express.json());
+app.use('/', risksRouter);
+
+describe('Risks', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('test1', async () => {
```
Comment (rishavsarma5) [inline src/backend/tests/risks.test.ts:17]: try to make the test names more descriptive as to what they are trying to do

File: src/backend/tests/risks.test.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import request from 'supertest';
+import express from 'express';
+import risksRouter from '../src/routes/risks.routes';
+import prisma from '../src/prisma/prisma';
+// import { Role } from '@prisma/client';
+// import { getChangeRequestReviewState, getHighestProjectNumber } from '../src/utils/projects.utils';
```
Comment (rishavsarma5) [inline src/backend/tests/risks.test.ts:6]: delete

File: src/backend/tests/risks.test.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  });
+
+  test('test2', async () => {
+    const res = await request(app).get('/123123123');
+    console.log(res.body);
+    expect(res.body.message).toBe('Project with id 123123123 not found!');
+    expect(res.statusCode).toBe(404);
+  });
+});
+console.log('hello');
```
Comment (rishavsarma5) [inline src/backend/tests/risks.test.ts:70]: delete

File: src/backend/tests/risks.test.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+          role: 'APP_ADMIN'
+        }
+      }
+    ]);
+    expect(res.statusCode).toBe(200);
+  });
+
+  test('test2', async () => {
+    const res = await request(app).get('/123123123');
+    console.log(res.body);
```
Comment (rishavsarma5) [inline src/backend/tests/risks.test.ts:65]: delete

File: .vscode/launch.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (rishavsarma5) [inline .vscode/launch.json:2]: @anthonybernardi is this file supposed to be changed? im not sure if it is necessary.

File: src/backend/tests/risks.test.ts [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+          email: 'emrax.t@husky.neu.edu',
+          emailId: 'emrax.t',
+          role: 'APP_ADMIN'
+        }
+      }
+    ]);
+    expect(res.statusCode).toBe(200);
+  });
+
+  test('test2', async () => {
```
Comment (rishavsarma5) [inline src/backend/tests/risks.test.ts:63]: see above comment above renaming tests

**Comment by rishavsarma5** [comment]
great job on your first pr! there are a few comments i left so take a look at those. also, i think you can convert this pr into ready for review after making those changes.

File: .vscode/launch.json (hunk 5 above)
Comment (anthonybernardi) [reply .vscode/launch.json:2]: definitely not

**Comment by anthonybernardi** [comment]
also no reason for the yarn.lock to change

**Review by anthonybernardi — CHANGES_REQUESTED** [review]
still need to mock all the prisma calls made -- check out how other tests do it or research a little online about what it means to mock a database, why we do it, how to mock with jest, things like that

**Pull request closed without merging**